src/app/loadout/plug-drawer/PlugDrawer.tsx

```diff
@@ -80,14 +85,14 @@ export default function PlugDrawer({
   const language = useSelector(languageSelector);
   const [query, setQuery] = useState(initialQuery || '');
   const [selected, setSelected] = useState(() =>
-    createInternalSelectedState(plugSets, initiallySelected)
+    assignPlugsToPlugSets(plugSets, initiallySelected)
   );
-  const filterInput = useRef<SearchFilterRef | null>(null);
+  const filterInput = useRef<HTMLInputElement>(null);
```

Comment:
This wasn't hooked up to anything, so I hooked it up to the input

Reply:
Hmm what is it used for if that is the case I wonder. I don't remember the point of it.